storage/aptosdb/src/lib.rs

```diff
@@ -413,19 +403,17 @@ impl AptosDB {
                     STATE_MERKLE_DB_NAME,
                     gen_state_merkle_cfds(&rocksdb_configs.state_merkle_db_config),
                 )?,
-                if rocksdb_configs.use_state_kv_db {
-                    Some(DB::open_cf(
-                        &gen_rocksdb_options(&rocksdb_configs.state_kv_db_config, false),
-                        state_kv_db_path.clone(),
-                        STATE_KV_DB_NAME,
-                        gen_state_kv_cfds(&rocksdb_configs.state_kv_db_config),
-                    )?)
-                } else {
-                    None
-                },
             )
         };
 
+        let ledger_db = Arc::new(ledger_db);
+        let state_kv_db = StateKvDb::new(
+            db_root_path,
+            rocksdb_configs,
+            readonly,
+            Arc::clone(&ledger_db),
+        )?;
+
         if rocksdb_configs.use_state_kv_db {
             info!(state_kv_db_path = state_kv_db_path, "Opened state K/V DB.",);
```

Comment:
* move these logging to `StateKvDb::new()`
* remove `state_kv_db_path`

Reply:
Done